crates/polars-json/src/ndjson/file.rs

```diff
@@ -88,8 +90,10 @@ impl<R: BufRead> FallibleStreamingIterator for FileReader<R> {
 
 fn parse_value<'a>(scratch: &'a mut Vec<u8>, val: &[u8]) -> PolarsResult<BorrowedValue<'a>> {
     scratch.clear();
-    scratch.extend_from_slice(val);
+    let bytes = remove_bom(val);
```

Comment:
I don't think we should do that here. It will do a check for every value and there should only be one `BOM`, that is at the start of the file.